### PR TITLE
Add Sample API Endpoints

### PR DESCRIPTION
**Pull request by lao9**
Hey @case-eee ! Jonathan and I just brainstormed some possible endpoints for our QS API. If you have the opportunity to review, let us know if you have any feedback. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+
+`GET /api/v1/diary`
+
+# Add Food to Meal
+
+`POST /api/v1/diary?food_id=[]&meal_id=[]`
+
+# Remove Food from Meal
+
+`DELETE /api/v1/diary?food_meal_id=id`
```
Comment (case-eee) [inline README.md]: wouldn't this be "/api/v1/diary?food_meal_id=[]" if you're using [] to represent a user's input?

**Review by case-eee — COMMENTED** [review]
@lao9 @jk1dd this is looking great 👍 🎉 left ya one comment!

File: README.md (hunk 1 above)
Comment (lao9) [reply README.md]: Thanks for taking a look @case-eee ! I updated it, you're correct, we just had a bit of an inconsistency there.

@jk1dd - do you want to merge this when you get a chance?

Comment (case-eee) [reply README.md]: 👍

**Pull request merged**